presto-docs/src/main/sphinx/release/release-335.rst

```diff
@@ -0,0 +1,42 @@
+=========================
+Release 335 (XX May 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix failure when :func:`reduce_agg` is used as a window function. (:issue:`3883`)
+* Fix incorrect cast from ``TIMESTAMP WITHOUT TIME ZONE`` to ``TIME`` type. (:issue:`3848`)
+* Improve optimized local scheduling with regard to non-uniform data distribution. (:issue:`3922`)
+* Add support for variable-precision ``TIMESTAMP WITHOUT TIME ZONE`` type. (:issue:`3783`)
```

Comment:
do we want to link to docs for that after the merge of the doc PR?

Reply:
yes, that's a good idea

Reply:
We should write this as

``TIMESTAMP`` (without time zone) type.

Since we never document or display the extra `WITHOUT TIME ZONE` syntax.